src/HierarchicalComboBox/HierarchicalComboBox.js

```diff
@@ -17,25 +16,24 @@ const HierarchicalComboBox = ({
   renderInputToolType,
   ...props
 }) => {
-  const optionsList = [];
-  getFormattedOptionsList(optionsList, values, 0, separator, maxCharLength);
 
   return (
     <Autocomplete
-        {...props}
+      id="combo-box-demo"
```

Comment:
We should remove the "combo-box-demo" id property

Reply:
Yes, I removed it. Thanks.